integrations/java/iceberg-1.2/openhouse-java-runtime/build.gradle

```diff
@@ -19,7 +19,7 @@ configurations {
 }
 
 ext {
-  icebergVersion = '1.2.0'
+  icebergVersion = '1.2.0.4'
```

Comment:
Can we have the iceberg 1.2.x version defined in a common place so that we don't need to change in every file?

Reply:
Done.